static/js/containers/ProfilePage.js

```diff
@@ -40,9 +39,7 @@ class ProfilePage extends React.Component {
       dispatch(startProfileEdit());
     }
     return dispatch(validateProfile(profile, requiredFields, messages)).then(() => {
-      dispatch(saveProfile(SETTINGS.username, profile)).then(() => {
-        dispatch(clearProfileEdit());
-      });
+      dispatch(saveProfile(SETTINGS.username, profile));
```

Comment:
@aliceriot  Actually the scenario is when user press ``save and continue`` on privacy tab it should take user to ``personal tab``.
- And it does this well.
- The issue is when it take user to ``personal tab`` after few milisec the highlighted errors gets remove.
- And my investigation shows that it is because of `` dispatch(clearProfileEdit());``.
- It seems, app is saving profile on my machine after it navigate to ``personal tab``. (I mean to say save request complete after few milli sec and at that time it is already on ``personal tab`` and edit mode gets clear.